mesonbuild/modules/rust.py

```diff
@@ -344,6 +344,11 @@ def bindgen(self, state: ModuleState, args: T.List, kwargs: FuncBindgen) -> Modu
         if are_asserts_disabled_for_subproject(state.subproject, state.environment):
             clang_args.append('-DNDEBUG')
 
+        time64 = state.get_option('b_time64', state.subproject)
+        assert isinstance(time64, bool), 'for mypy'
+        if time64:
+            clang_args.append('-D_TIME_BITS=64')
```

Comment:
We had a recent change here that with the `_FILE_OFFSET` bits maybe we can add this to the same block? (it uses the CCompiler instance so we avoid the need to copy the define here)